src/app/clusters/general-diagnostics-server/general-diagnostics-server.cpp

```diff
@@ -17,6 +17,8 @@
 
 #include "general-diagnostics-server.h"
 
+#include <zap-generated/gen_config.h>
```

Comment:
That said, the normal way this gets included is via `app/util/config.h`.  The fact that generated config is involved is an implementation detail.... @cecille